tools/protoc-gen-spireplugin/main.go

```diff
@@ -30,7 +32,11 @@ func main() {
 	req := new(plugin.CodeGeneratorRequest)
 	panice(proto.Unmarshal(reqData, req))
 
-	g := NewGenerator(req)
+	// parse parameters
+	options := parseOptions(req.GetParameter())
+	options = options
```

Comment:
Out of curiosity, what does this operation do?

Reply:
Oops. That's development cruft left to silence the "unused" compiler error (before i had plumbed through options). Will remove.